openstack/compute/v2/extensions/evacuate/results.go

```diff
@@ -0,0 +1,11 @@
+package evacuate
+
+import (
+	"github.com/gophercloud/gophercloud"
+)
+
+// EvacuateResult is the response from an Evacuate operation. Call its ExtractErr
+// method to determine if the request suceeded or failed.
```

Comment:
This needs updated to say "Call its ExtractAdminPass method to retrieve the admin password of the instance. The admin password will be an empty string if the cloud is not configured to inject admin passwords"

And the lines should be wrapped at 80-characters.